conf/mod_premium_lib.conf.dist

```diff
@@ -0,0 +1,9 @@
+[worldserver]
+
+#
+# Duration:
+# How many days the player will be premium. If set to 0, the vip will never expire.
```

Comment:
```suggestion
# Set the default duration of the premium status when using ".premium account create" or ".premium character create" . If set to 0, the premium status will never expire.
```

Reply:
ofc if command names are changed as i suggested, hit me up and i'll change the description